### PR TITLE
Ru/dcos 60843

### PR DESCRIPTION
**Pull request by russurquhart1**
## Jira Ticket
Before creating this pull request, please make sure you have a separate ticket for the docs team to track their work to review and merge, even for minor edits. If you need assistance, ask in the #doc-team channel on slack.

https://jira.mesosphere.com/browse/DCOS-60843

## Description of changes being made
Moved sections and information from Kommander Quick Start page to more relevant areas of the documentation.

## Checklist
- [ ] Make sure you change all affected versions (e.g. 1.10, 1.11, 1.12, 1.13).
- [ ] Test all commands and procedures where applicable.
- [ ] Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects) if you are moving a page.

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing) for more information.

### CODE REVIEW

**Comment by russurquhart1** [comment]
Would like to add Kommander screenshot.

File: pages/ksphere/kommander/latest/quick-start/index.md [hunk 1]
```diff
@@ -31,7 +33,7 @@ Before starting the Konvoy installation, you should verify the following:
 
 ### Download and Install
 
-Download Konvoy with Kommander [here](https://github.com/mesosphere/konvoy/releases).
+Download [Konvoy with Kommander](https://github.com/mesosphere/konvoy/releases).
 
 Download the tarball to your local Downloads directory.
```
Comment (cmastr) [inline pages/ksphere/kommander/latest/quick-start/index.md]: Aren't these the same instructions from Konvoy at this point? If so, we should remove this whole section and point them to the Konvoy docs.

**Comment by ck4adventure** [comment]
I recommended using this whenever you are ready to merge: https://github.com/mesosphere/dcos-docs-site/pull/2692/

**Comment by cmastr** [comment]
closing as this PR has been replaced.

**Pull request closed without merging**